src/drawline.c

```diff
@@ -1698,20 +1695,9 @@ win_line(
 	{
 	    wlv.vcol -= charsize;
 	    ptr = prev_ptr;
-	    // If the character fits on the screen, don't need to skip it.
-	    // Except for a TAB.
-	    if (((*mb_ptr2cells)(ptr) >= charsize || *ptr == TAB)
-							       && wlv.col == 0)
-	       n_skip = v - wlv.vcol;
```

Comment:
Is that correct to remove? What happens with TAB at the end of a screen-line?

Reply:
I don't think I "removed" anything here. I just renamed `n_skip` to `skip_cells`.

Reply:
Also tests added in patch 8.0.0394 should cover this.

Reply:
Also added another test than only works when `skip_cells` and `n_skip` are combined.